### DISCUSSION TITLE
Does Stash Save MetaData from pictures?

### DISCUSSION THREAD

**Discussion by pto199** [stashapp/stash#4305] [Q&A] [answered]
I had a bunch of A.I. Generated pictures I had on my computer a long time ago that I deleted.  I thought they were gone forever but recently opened up Stash and noticed they're all in the stash database.  Of course the actual files are missing so all I can see is the filename and a thumbnail thats left over as I havent cleaned the stash database yet.  

But I cant regenerate the images because, even tho the generation seed is stored in the filename.... the filenames are cutoff as the prompt is really long.  Also I dont know which model I used.  All that information (Full prompt, negative prompts, model used, etc.... would be in the metadata of the original A.I. picture.  Does Stash save this metadata somewhere?  I can't see it anywhere but figured I would ask here in case it is stored somewhere

**Comment by dingdongsolong4** [comment, marked as answer]
If you're referring to metadata that is stored in the image file itself (ie like Location, Camera settings, etc), then unfortunately not. Stash only stores the height, width and file format of image files in the database.
